core/src/main/java/org/apache/cxf/catalog/OASISCatalogManager.java

```diff
@@ -100,10 +100,10 @@ public String getResolvedEntity(String publicId, String systemId) {
                     String s = super.getResolvedEntity(publicId, systemId);
                     if (s != null && s.startsWith("classpath:")) {
                         try {
-                            URIResolver r = new URIResolver(s);
-                            if (r.isResolved()) {
-                                r.getInputStream().close();
-                                return r.getURL().toExternalForm();
+                            try (URIResolver r = new URIResolver(s)) {
```

Comment:
Do we really need to have a nested try statement here?

Reply:
No, not really in this case, simplified, thanks!